README.md

```diff
@@ -138,6 +138,19 @@ From the root of the project:
 
 Once the sever has started, the webapp is accessible at [https://security-hq.local.dev-gutools.co.uk/](https://security-hq.local.dev-gutools.co.uk/)
 
+### DynamoDB local setup
+
+Running Security HQ locally requires a local instance of DynamoDb to connect
+to. The `run` tasks handles most of the work automatically by starting and
+stopping `DynamoDBLocal` as appropriate. It does not, however, create a table
+for the application to use.
+
+To do that, the following script needs to be executed _while_ the application
+is running. This only needs to be done once, as the database is stored on disk
+and persistent between application runs.
+
+1. `$ ./script/setup`
```

Comment:
I do find this workflow a little unusual, but we can review again once we've taken a look at integration tests and/or other ways to run dynamo locally.

Reply:
Yes, I agree. I think the way this has been setup allows us to iterate on the approach fairly easily.